tests/test_run_feature_vector.py

```diff
@@ -0,0 +1,57 @@
+import os
+import numpy as np
+from ipfx.bin.run_feature_vector_extraction import run_feature_vector_extraction
+from ipfx.bin.run_feature_collection import run_feature_collection
+import pandas as pd
+import pytest
+import ipfx.bin.lims_queries as lq
+
+
+TEST_OUTPUT_DIR = "/allen/aibs/informatics/module_test_data/ipfx/test_feature_vector"
+
+if not lq.able_to_connect_to_lims():
+    pytest.skip("cannot connect to LIMS", allow_module_level=True)
```

Comment:
I think this is probably okay. 

But if you think you will be adding more lims required tests in the future, it would probably be more extensible if you marked tests that requires lims with `@pytest.mark.requires_lims` and then in your conftest.py apply the skip to to all tests with the `requires_lims` mark if you can't make a connection to lims.

Reply:
@njmei Good suggestion. For some silly reasons I will add your suggestions as a part of a separate PR today.